engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/Builtin.java

```diff
@@ -0,0 +1,4 @@
+package org.enso.interpreter.node.expression.builtin;
+
+/** A base class for all classes annotated with @BuiltinType. Temporarily a placeholder. */
```

Comment:
Why is this a placeholder? What are the plans for it?

Reply:
Placeholder is maybe a bad word. Needed a base class for all those builtin types but it will also probably expand as part of https://github.com/enso-org/enso/issues/1239